### PR TITLE
UI Fixes

### LINKED ISSUE #84: Create popup window for when coop campaign is clicked
This menu should include (Host New Game, Host Saved Game, Direct Connect). The wording and design of these buttons is up to the developer.

### PR DESCRIPTION
**Pull request by garrettluskey**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] All new classes have class-level documentation comments, if there are any at all
- [x] Tests for the changes have been added (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation update
- [ ] Other... Please describe:

## What is the current behaviour?

Currently the host UI and join UI were not working

## What is the new behaviour?

This PR fixes the UI issues. The Host Game UI now hosts a game properly. Join Game now joins properly. Keep in mind when connected to a local server to use "localhost" or "127.0.0.1"

## Other information

### CODE REVIEW

File: source/Coop.Core/CoopartiveMultiplayerExperience.cs [hunk 1]
```diff
@@ -21,11 +24,37 @@ public CoopartiveMultiplayerExperience()
...
+            MessageBroker.Instance.Unsubscribe<HostSave>(Handle);
+        }
+
+        private void Handle(MessagePayload<ConnectWithIP> obj)
+        {
+            var connectMessage = obj.What;
+
+            var config = new NetworkConfiguration()
+            {
+                Address = connectMessage.Address.ToString(),
```
Comment (EgardA) [inline source/Coop.Core/CoopartiveMultiplayerExperience.cs:46]: Address in debug is read only, fails to build

**Review by EgardA — COMMENTED** [review]
Did you get this to run in Debug? 

Buttons are working but nothing happens on my end.

File: source/Coop.Core/Server/States/InitialServerState.cs [hunk 2]
```diff
@@ -34,8 +34,7 @@ internal void Handle_GameLoaded(MessagePayload<CampaignReady> payload)
 
     public override void Start()
     {
-        // TODO use UI screen
-        MessageBroker.Publish(this, new LoadDebugGame());
```
Comment (EgardA) [inline source/Coop.Core/Server/States/InitialServerState.cs:38]: Add this back

Comment (garrettluskey) [reply source/Coop.Core/Server/States/InitialServerState.cs:38]: Done

File: source/Coop.Core/CoopartiveMultiplayerExperience.cs (hunk 1 above)
Comment (garrettluskey) [reply source/Coop.Core/CoopartiveMultiplayerExperience.cs:46]: Fixed

**Review by EgardA — APPROVED** [review]

**Pull request merged**